daemon/web/components/TerminalView/index.sass

```diff
@@ -2,14 +2,16 @@
 
 .terminal-view
   box-sizing: border-box
-  height: 100%
+  height: 496px
+  width: &-width
+  vertical-align: middle
 
   > .text-area
     background: $darkgrey
     color: $white
-    font-size: 15px
+    font-size: 14px
     height: 100%
-    padding: 8px 38px
+    padding: 8px 30px
```

Comment:
can you change this to use the new `spacing.sass` [padding classes](https://github.com/ubclaunchpad/inertia/blob/master/daemon/web/styles/spacing.sass#L10) instead?

Reply:
in this case, you might make a new spacing definition:

```sass
$spacing-xxs: 8px
```

then a new top/bottom class (if `padding` has 2 values, the first value is top/bottom, second is sides)

```sass
  &-top, &-ends
    &-xxs
      padding-top: $spacing-xxs

  &-bottom, &-ends
    &-xxs
      padding-bottom: $spacing-xxs
```

since there's no class for `spacing-s` yet (30px), you'll want to make one:

```sass
  &-left, &-sides
    &-s
      padding-left: $spacing-s

  &-right, &-sides
    &-s
      padding-right: $spacing-s
```

then you can:

```html
<div className="pad-ends-xxs pad-sides-s"> ... </div>
```